### PR TITLE
[UI] NavBar SubOption Style Fix #1433

### LINKED ISSUE #1433: [UI] Width Fix for Suboptions in Navbar
### Description

Navlinks on the navbar are uneven. Checkout the below screenshot, the option has a logo and text however the logo and text doesn't maintain the same width in every option. Hovering over them can highlist the issue.

image

****
image

### Expected Behavior

We can make the logo and text hold certain width so that all the option has uniform width.

### Screenshots

image

image

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Abbhiishek**
**Description**

This PR fixes #1433 

**Old Style** 

https://github.com/meshery/meshery.io/assets/86338762/2bb0195f-6982-4fb7-ba85-407aeb9cee9e

**New Style**

https://github.com/meshery/meshery.io/assets/86338762/230e4761-1822-4bde-a245-ea5bbcfd09ec

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by VivekBaldha0712 — COMMENTED** [review]
LGTM !!

**Comment by Abbhiishek** [comment]
Hey @VivekBaldha0712 

what i have to do next for getting this pr merged under Hacktoberfest ?

Thanks

**Comment by saurabh100ni** [comment]
Let's discuss this in tomorrow's website meeting,
Please add it in the Doc, If you would ^^:
https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit

**Comment by leecalcote** [comment]
> LGTM !!

I agree.

File: _sass/navigation.scss [hunk 1]
```diff
@@ -42,7 +42,7 @@ nav {
 .nav-item:not(.has-sub) > a:not(.highlight).nav-text{
 	position: relative;
 	&:hover{
-		background-image: var(--background-grey);
+		// background-image: var(--background-grey);
```
Comment (leecalcote) [inline _sass/navigation.scss:45]:
```suggestion
```

File: _sass/navigation.scss [hunk 2]
```diff
@@ -121,7 +121,7 @@ a.nav-text:hover {
 		}
 	}
 	&:hover{
-		background-image: var(--background-grey);
+		// background-image: var(--background-grey);
```
Comment (leecalcote) [inline _sass/navigation.scss:124]:
```suggestion
```

**Review by leecalcote — DISMISSED** [review]

**Comment by leecalcote** [comment]
Thanks, @VivekBaldha0712. Good work, @Abbhiishek 👍

**Pull request merged**